test/cypress/integration/my-services/my-services-notification-banner.cy.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+const userStubs = require('../../stubs/user-stubs')
+
+const userExternalId = 'authenticated-user-id'
+
+describe('Service has a live account that supports payouts', () => {
```

Comment:
```suggestion
describe('My services notification banner or something ?', () => {
```

Reply:
good spot - copy and paste we didn't change

Reply:
updated